stubs/pydbus/pydbus/proxy.pyi

```diff
@@ -1,13 +1,142 @@
-from typing import Generic, TypeVar, type_check_only
+from typing import Any, Generic, Literal, TypedDict, TypeVar, overload, type_check_only
 from typing_extensions import Self
 from xml.etree.ElementTree import Element
 
 from .bus import Bus
+from .generic import bound_signal
 
 _T = TypeVar("_T")
+_ST = TypeVar("_ST")  # signal type
 
-class ProxyMixin(Generic[_T]):
-    def get(self, bus_name: str, object_path: str | None = None, *, timeout: int | None = None) -> _CompositeObject[_T]: ...
+@type_check_only
+class OrgBluezAdapter1Dict(TypedDict, total=False):
```

Comment:
I'm sorry I didn't notice this earlier, but type check only types should be prefixed with `_`, e.g. `_OrgBluezAdapter1Dict`.

Reply:
I had to add `# noqa: Y049` for these types. Ruff does not see these as used even though they are getting passed into `TypedDict()`.

Reply:
How do I fix this issue? https://github.com/python/typeshed/actions/runs/14883678415/job/41797568758?pr=13921#step:5:317

Reply:
You can add it to the stubtest allowlist